### PR TITLE
Adding Rise In bootcamp community repos

### PR DESCRIPTION
**Pull request by Rise-In**

### CODE REVIEW

**Comment by Rise-In** [comment]
Adding BNB Chain projects

**Comment by Rise-In** [comment]
Added Internet Computer repos and deleted 2 duplicate repos

File: data/ecosystems/b/bnb-chain-bsc.toml [hunk 1]
```diff
@@ -45419,3 +45419,175 @@ url = "https://github.com/zzzuhaibmohd/DeFiHackLabs"
...
+url = "https://github.com/Ahmetakaslan/Contract_with_ui"
+
+[[repo]] 
+url = "https://github.com/MeryemSulum/Cargo_Tracking_Dapp"
+
+[[repo]] 
+url = "https://github.com/adeuSH1/BNB_bootcamp"
+
+[[repo]] 
+url = "https://github.com/srhtdmrkl/smart-tenders"
```
Comment (eherrerosj) [inline data/ecosystems/b/bnb-chain-bsc.toml:45574]: can you point out how's this built / deployed using BSC?

File: data/ecosystems/b/bnb-chain-bsc.toml [hunk 2]
```diff
@@ -45419,3 +45419,175 @@ url = "https://github.com/zzzuhaibmohd/DeFiHackLabs"
...
+url = "https://github.com/SpandanM110/Venture-Pitches"
+
+[[repo]] 
+url = "https://github.com/14metehan53/bnb-chain-yarisma"
+
+[[repo]] 
+url = "https://github.com/abdulhakim-altunkaya/solidity_turbo_staking"
+
+[[repo]] 
+url = "https://github.com/berkaykarul/sushiswap"
```
Comment (eherrerosj) [inline data/ecosystems/b/bnb-chain-bsc.toml:45535]: sushiswap fork, remove

File: data/ecosystems/b/bnb-chain-bsc.toml [hunk 3]
```diff
@@ -45419,3 +45419,175 @@ url = "https://github.com/zzzuhaibmohd/DeFiHackLabs"
...
+url = "https://github.com/shivam7868-alt/SOLIDITY-AND-BNB-CHAIN-DEVELOPMENT-BOOTCAMP-MARKETPLACE"
+
+[[repo]] 
+url = "https://github.com/LakshaySK106/BNB-PHR"
+
+[[repo]] 
+url = "https://github.com/AnuragNishad12/BNB_Project_2"
+
+[[repo]] 
+url = "https://github.com/AdityaCubicles45/Gitcode-v2"
```
Comment (eherrerosj) [inline data/ecosystems/b/bnb-chain-bsc.toml:45520]: can see Ethereum, Polygon and Avalanche, but not it using BSC (?)

**Review by eherrerosj — CHANGES_REQUESTED** [review]
lots of duplicates, can you please remove them? List is here:
https://github.com/electric-capital/crypto-ecosystems/actions/runs/7310009086/job/19918211744?pr=843#step:5:55

**Pull request closed without merging**